samples/xbee/communication/relay_frames_callback/README.md

```diff
@@ -0,0 +1,93 @@
+Relay Frame Callback Sample Application
+========================================
+
+This example demonstrates the usage of the User Data Relay frame API by giving
+an example of how to handle a relay frame and send a response back to the source
+interface within a callback.
+
+The example registers a callback to fire whenever a relay frame is received.
+This callback sends a response containing the message back to the source
+interface.
+
+Requirements
+------------
+
+To run this example you need:
+
+* One XBee3 radio module with MicroPython support.
+* One carrier board for the radio module (XBIB-U-DEV or XBIB-C board).
+* The XCTU application (available at www.digi.com/xctu).
```

Comment:
Should we list a minimum version with support for Relay Frames?

Reply:
This sample is specifically for relay frame callbacks, and the minimum version support is listed in the `Supported platforms` section.

Reply:
Sorry for not being more specific -- "minimum XCTU version" with support for generating Relay Frames as described in step 5 of "Run".  E.g. "XCTU application v1.2.3.4 or later (available at www.digi.com/xctu)".  And since this is a markdown file, use the "link" formatting for the URL on the text "XCTU application".

Reply:
Gotcha. Yeah I can do that.